### ISSUE TITLE
Test Individual Subject Page Interactive Elements for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4900]
# Page to Test:

Individual Subject - https://openlibrary.org/subjects/architecture

# Steps

Steps for this task are outlined in #4897
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by Yashs911** [comment]
@bpmcneilly I would like to work on this.

**Comment by Yashs911** [comment]
The search button does not have visual focus so I am thinking of adding a `:focus` selector with `border: 2px solid @green;`
![image](https://user-images.githubusercontent.com/64412143/113259687-098ae700-92eb-11eb-8e61-3402105eb253.png)

**Comment by bpmcneilly** [comment]
@Yashs911 - By default, most buttons should receive a visual indication of keyboard focus by default, so if this doesn't have one, my first guess would be to look for our existing CSS rules to see if we're doing anything to stop the indicator.

Most browsers use the `outline` property on focus to create the focus indicator, so declarations like `outline: 0` or `outline: none` will prevent focus indicators from appearing.

Even ff there isn't anything like that in our code, I would avoid using borders as focus indicators. Most buttons already have borders & we want to avoid using a color change as the primary indication of focus. I'd recommend adding an `outline` instead of the border, and if you need to you can use an `outline-offset` to make sure that the outline doesn't hug the existing border of the button too closely.

Let me know what you think here. If that's the only issue you found, you can close this!

**Comment by Yashs911** [comment]
@bpmcneilly no other issues according to me. As you rightly pointed out there is `outline: 0;` declaration.
![image](https://user-images.githubusercontent.com/64412143/113432466-10088400-93fb-11eb-905f-cd2785dfaa36.png)

**Comment by bpmcneilly** [comment]
Ok, awesome. Looking at that selector (just all `button` elements on the site), getting a PR that removes this will do a lot for other elements on the site!
